From d3c098c2fde55ddf0c7d56eae56925103d35da73 Mon Sep 17 00:00:00 2001
From: Romain Naour <romain.naour@gmail.com>
Date: Sat, 13 May 2017 18:56:51 +0200
Subject: [PATCH] test: add missing header

test/attest.c: In function 'at_command':
test/attest.c:43:2: error: unknown type name 'fd_set'
  fd_set rfds;
  ^

Fixes:
http://autobuild.buildroot.net/results/06c/06c930d9c5299b79500d018ac3fb2861ce834c7c/

Signed-off-by: Romain Naour <romain.naour@gmail.com>
---
 test/attest.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/test/attest.c b/test/attest.c
index 12ba682..2626cf1 100644
--- a/test/attest.c
+++ b/test/attest.c
@@ -35,6 +35,8 @@
 #include <sys/ioctl.h>
 #include <sys/socket.h>
 
+#include <sys/select.h>
+
 #include <bluetooth/bluetooth.h>
 #include <bluetooth/rfcomm.h>
 
-- 
2.9.3

